### DISCUSSION TITLE
Cleaner UI for IP and SSO notifications

### DISCUSSION THREAD

**Discussion by jmreid** [community/community#83086] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm in a few orgs that have IP address and/or SSO  restrictions. Whenever I use my Github account from a computer that isn't able to sign into these Orgs (which is ok), I have to live with fairly large banners across the top of the Home and Notification pages. Is there a way to make these more concise or maybe putting them into a single UI element? I understand that it is important for people to see them as they might miss important notifications, but there current state is pretty distracting.

<img width="310" alt="Screenshot 2023-12-29 at 8 22 39 PM" src="https://github.com/community/community/assets/76880/b33d43c0-b1db-4d9d-bfdb-1a9e6f26f49c">

**Comment by SimplyDanny** [comment]
I'd also like to be able to disable these banners entirely. They are distracting and you cannot do anything to get rid of them if you're unable to use SSO from a private device.

Something like "Don't show again", "Accept", "Ignore" or "Remind me later" or settings to configure the behavior are preferred from my side.

**Comment by jmreid** [comment]
Thank you for cleaning this up! 
<img width="285" alt="image" src="https://github.com/user-attachments/assets/21c8592b-9698-45f3-81d8-76a5014181aa">
